localeval.js

```diff
@@ -1,6 +1,10 @@
 // Thaddée Tyl <thaddee.tyl@gmail.com>. License: CC-BY v3.
 "use strict";
 var node_js = typeof exports === 'object';
+if ("undefined" !== typeof process && process.browser) {
+    // webpack sets browser property on process global
+    node_js = false;
+}
 (function (root, factory) {
   if (typeof exports === 'object') {
     // Node.
```

Comment:
This is the commonjs section of your UMD module. It'll also be run for browserify and anything else that executes or bundles in a commonjs environment. I think it's fine to keep it platform-agnostic, but again, just my opinion.